tests/test.txt

```diff
@@ -80,16 +80,16 @@ tests/testSpline.py
 #tests/testStatistics.py
```

Comment:
Unfortunately many tests call `checkPickle`, which is what segfaults. I prefer to leave the test commented out for now, but it's easy to tweak it so it runs by adding `return` as the first statement of `checkPickle`. If I do that the test runs, so I'm confident it can be enabled when pickling is fixed.